### PR TITLE
Add a first pass at a coverage lens for spyglass

### PR DESCRIPTION
**Pull request by Katharine**
This PR adds a coverage lens to Spyglass, and enables it for our e2e conformance coverage job. In the future we will want to link to line coverage as well (quite probably by using go's built in generation and linking to that). You can hover over a square to see the filename and statement coverage for it.

It looks like this:

image

/assign @spiffxp

### CODE REVIEW

**Comment by Katharine** [comment]
Actually come to think of it, we have more people doing web things now! Comments appreciated.

/cc @mirandachrist

File: prow/spyglass/lenses/coverage/coverage.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      node.style.left = `0`;
+    }
+    offset += percentage;
+    if (child.totalFiles === 1) {
+      node.classList.add('leaf');
+      const [filename, file] = child.files.entries().next().value;
+      node.title = `${filename}: ${(file.coveredStatements / file.totalStatements * 100).toFixed(0)}%`;
+      node.style.backgroundColor = NO_COVERAGE.mix(FULL_COVERAGE, file.coveredStatements / file.totalStatements).hex();
+      // Not having a border looks weird, but if we use a constant colour then some parts of this
+      // end up only having
```
Comment (michelle192837) [inline prow/spyglass/lenses/coverage/coverage.ts]: nit: Clarify?

Comment (michelle192837) [reply prow/spyglass/lenses/coverage/coverage.ts]: Sorry; nit: Can you clarify this comment? Not sure if something was left off the end or I'm not parsing it right.

Comment (Katharine) [reply prow/spyglass/lenses/coverage/coverage.ts]: I think I forgot to finish typing it.

**Comment by michelle192837** [comment]
Sweet, looks good to me.
/lgtm

**Pull request merged**